src/map/skill.c

```diff
@@ -4781,18 +4789,22 @@ static int skill_castend_damage_id(struct block_list *src, struct block_list *bl
 				if (bl->id==skill->area_temp[1])
 					break;
 				if (skill->attack(BF_WEAPON,src,src,bl,skill_id,skill_lv,tick,SD_ANIMATION))
-					skill->blown(src,bl,skill->area_temp[2],-1,0);
+					skill->blown(src, bl, skill->area_temp[2], UNIT_DIR_UNDEFINED, 0);
 			} else {
-				int x=bl->x,y=bl->y,i,dir;
-				dir = map->calc_dir(bl,src->x,src->y);
+				int x=bl->x,y=bl->y,i;
+				enum unit_dir dir = map->calc_dir(bl,src->x,src->y);
```

Comment:
Missing spaces in argument list: `map->calc_dir(bl, src->x, src->y)`

Reply:
Done.